app/src/nxt/res/layout/fragment_nxt_firmware.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In my opinion the dialog has enough space for users.
But I found the other preferences are collapsed in Nexus 5X emulator, particularly sensors and thresholds.
So we need to fix them in another pull-request.

For this PR, please merge it.